rollup.config.js

```diff
@@ -26,7 +24,7 @@ const configurator = (file, format, plugins = []) => ({
       regl: 'createREGL',
     },
   },
-  plugins: [...basePlugins, ...plugins],
+  plugins: [...basePlugins(), ...plugins],
```

Comment:
Yeah, sorry for not clarifying.

> Do we not need commonjs anymore? 

I don't think we need `commonjs` anymore because all the bundled deps (not `external`) have a `module` field.

> why was `basePlugins` changed the way it was changed?

I'm having trouble finding the relevant issue, but rollup plugins may have internal state and it's recommended to not reuse instances across different builds. I changed `basePlugins` to a function so that new instances are returned for each individual config. Probably not necessary, but a good practice.

Reply:
Thanks for the clarifications!